interpreter/program.go

```diff
@@ -83,7 +84,22 @@ func NewProgram(expression *exprpb.Expr,
 	return &exprProgram{
 		expression:      expression,
 		revInstructions: revInstructions,
-		metadata:        newExprMetadata(info)}
+		metadata:        newExprMetadata(info),
+		forceEval:       false,
+	}
+}
+
+// NewCompleteProgram creates a Program from a CEL expression and source
+// information which force evaluating all branches of the expression.
+func NewCompleteProgram(expression *exprpb.Expr,
```

Comment:
Maybe call this `NewExhaustiveProgram`. The word `Complete` connotes that somehow the regular program is incomplete.

Reply:
Done.